crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -141,6 +154,23 @@ impl EthTransactionRequest {
                     access_list: access_list.unwrap_or_default(),
                 }))
             }
+            // op-stack deposit
+            (Some(126), _, None, None, None) => {
```

Comment:
I think we should also require None here because deposit tx dont' have a price field, right?